linera-base/build.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() {
+    cfg_aliases::cfg_aliases! {
+        web: { all(target_arch = "wasm32", feature = "web") },
+        with_metrics: { all(not(target_arch = "wasm32"), feature = "metrics") },
```

Comment:
I think this could be risky if we ever try to add some sort of metrics to Wasm. Would it be possible to use just the feature flag?

```suggestion
        with_metrics: { feature = "metrics" },
```

Reply:
As discussed offline, this wouldn't work with `cargo test --all-features`.

Reply:
Not just `--all-features` — we would have to worry about (dependencies of!) sibling packages breaking the build.

If we could just use `feature = "metrics"` then there would be no need for all this `cfg_aliases` chicanery :)